util/queue.py

```diff
@@ -66,9 +71,20 @@ class Queue(multiprocessing.queues.Queue):
     """
 
     def __init__(self, *args, **kwargs):
+        if sys.version_info >= (3, 4) and 'ctx' not in kwargs:
```

Comment:
Please add a comment here explaining why this is necessary (i.e. why we need multiprocessing.get_context() in Python 3.4+).